app.rb

```diff
@@ -0,0 +1,96 @@
+require_relative 'book'
+require_relative 'item'
+require_relative 'label'
+require_relative 'books_ui'
+require_relative 'game/game_ui'
+require_relative 'music_album'
+require_relative 'genre'
+require 'json'
+
+class App
+  attr_accessor :books, :labels, :game
+
+  def initialize
+    @books = BooksUI.new
+    @game = GameUI.new
+    @genres = []
+    @music_albums = []
+  end
+
+  def save_data
+    @books.save_books
+    @books.save_labels
+    save_json_data('./data/genres.json', @genres)
+    save_json_data('./data/music_albums.json', @music_albums)
+  end
+
```

Comment:
- [ ] Please ensure that a user exit the app without any errors and the data is preserved correctly in a `.json` file so that the data can be loaded when the app starts again :pray: Currently, I got this error when the exit the application :point_down: so make sure that you fix this issue :pray: 
  
  ![image](https://github.com/QueenterJuma/ruby_catalog/assets/92303891/7f83168d-fe85-4244-b76c-a0ce9645ae58)

Reply:
Done. Thanks